sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -17,8 +17,8 @@
 
 
```

Comment:
Could we add a docstring to this function as well?
```suggestion
"""
Converts datetime to seconds since the Epoch in UTC.

:param input_datetime: Input datetime
:type input_datetime: datetime
:return: Integer
:rtype: int
"""
```